### PR TITLE
carousel: switch to string constants

### PR DESCRIPTION
**Pull request by XhmikosR**
The logic is the same, minus the `toUpperCase()` call. Should have done this in #30358.

@patrickhlauke I assume `event.pointerType` are always lowercase, right?

### CODE REVIEW

**Review by rohit2sharma95 — APPROVED** [review]

File: js/src/carousel.js [hunk 1]
```diff
@@ -278,7 +276,7 @@ class Carousel extends BaseComponent {
 
   _addTouchEventListeners() {
     const start = event => {
-      if (this._pointerEvent && PointerType[event.pointerType.toUpperCase()]) {
+      if (this._pointerEvent && (event.pointerType === POINTER_TYPE_PEN || event.pointerType === POINTER_TYPE_TOUCH)) {
```
Comment (patrickhlauke) [inline js/src/carousel.js:279]: how about defining POINTER_TYPE_MOUSE and checking here simply that it's NOT the mouse?

but yes seems kosher even as it is

**Review by patrickhlauke — APPROVED** [review]

**Comment by patrickhlauke** [comment]
and yes, the spec defines these as lowercase values https://w3c.github.io/pointerevents/#dom-pointerevent-pointertype

**Comment by patrickhlauke** [comment]
sorry, wrong button press...embarassing

File: js/src/carousel.js (hunk 1 above)
Comment (XhmikosR) [reply js/src/carousel.js:279]: I thought I'd go with the exact same checks, but we can change it later if needed :)

**Pull request merged**